app/code/Magento/Catalog/Model/Product/Price/PricePersistence.php

```diff
@@ -206,13 +214,13 @@ private function getAttributeId()
      */
     private function retrieveAffectedIds(array $skus)
     {
-        $affectedIds = [];
+        $affectedIds = [[]];
```

Comment:
Do we need a multi-dimensional array here?

Reply:
It's needed for getting rid of array_marge in the loop, but we can actually have there just  `$affectedIds = [];`, and below something like ` $affectedIds = array_merge([], ...$affectedIds);` that will look a bit better